src/main/java/ru/spbstu/reader/VcfReader.java

```diff
@@ -0,0 +1,27 @@
+package ru.spbstu.reader;
+
+import htsjdk.variant.vcf.VCFIterator;
+import htsjdk.variant.vcf.VCFIteratorBuilder;
+import ru.spbstu.reader.dto.VcfRecord;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class VcfReader {
+
+    public static List<VcfRecord> read(final InputStream inputStream) throws IOException {
+        return read(new VCFIteratorBuilder().open(inputStream));
+    }
+
+    public static List<VcfRecord> read(final String path) throws IOException {
+        return read(new VCFIteratorBuilder().open(path));
+    }
+
+    private static List<VcfRecord> read(final VCFIterator iterator) {
+        return iterator.stream()
+                .map(VcfRecord::from)
+                .collect(Collectors.toList());
+    }
```

Comment:
Maybe it would be better to return just a `Stream` because in some cases to read all records eagerly can be very expensive in terms of memory. Also using `Stream` is more flexible.

Reply:
It would be enough to use just the first one for now.